canonical-websites/www.ubuntu.com
# Set up D3 and create basic JS helper functions for desktop stats page

## Done

- Renamed desktop-statistics.js to stickyNav.js (and left as-is)
- Added a dummy data file for testing before hooking up to Grafana API
- Added a class that makes the charts always appear below the title+text on mobile

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/statistics
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that the sticky nav still works
- Check that the cards with percentage values in them have been calculated from the dummy data (you can test by editing the values in dummyData.js)
- Check that on mobile the charts always appear below the title+text
